### DISCUSSION TITLE
Subroutine & Function & Method & Procedure

### DISCUSSION THREAD

**Discussion by f1v3-dev** [NHN-Team-03/ITP#6] [General]
이 넷의 차이는 무엇일까요?

**Comment by P-C-Space** [comment]
Function > method = procedure = subroutine

**Comment by P-C-Space** [comment]
다만 subroutine은 function을 의미하지만 routine의 하위 요소들이다.

**Comment by f1v3-dev** [comment]
Method와 Procedure는 function을 구현한 하나의 방법 !

**Reply by f1v3-dev** [reply]
수학에서의 _function_ 이라는 개념을 바꾸려고 하지 말자

**Comment by damho-lee** [comment]
function을 언어들에서 자기들 말로 바꾼것
